addons/l10n_tr/data/account_tax_template_data.xml

```diff
@@ -35,16 +35,16 @@
         ]"/>
     </record>
 
-    <record id="tr_kdv_satis_purchase_18" model="account.tax.template">
+    <record id="tr_kdv_satis_purchase_20" model="account.tax.template">
```

Comment:
same as above, create a new tax with the same details (just changing the 18% to 20%), and add a field to the old 18% tax with `<field name="active" eval="False"/>`